rholang/src/main/scala/coop/rchain/rholang/interpreter/PrettyPrinter.scala

```diff
@@ -286,6 +289,23 @@ case class PrettyPrinter(
         }
     }
 
+  private def buildSendSeq[T <: GeneratedMessage](s: Seq[T]): Coeval[String] = {
```

Comment:
I don't think this should be an exceptional case. We should be doing the `isName` check whenever we're referencing a variable.

For example, parsing and printing this should return the original code:
```
new x in { *x }
```
but I think it won't because we do an exception for send arguments only.

I think we shouldn't be doing any exceptions here. Sp no `buildSendSeq` method, just the `isName` one. Unless you've seen a case where we should and I'm missing sth?